### ISSUE TITLE
move away from flit

### ISSUE THREAD

**Issue by ddkohler** [yaq-project/yaqc-qtpy#4]
@untzag :
> I believe we must move away from flit and use setuptools instead.
Flit does not support data files [1] which means we will never be able to distribute .desktop and .bat files and place them in the appropriate place for the operating system to use.
[1] https://github.com/pypa/flit/issues/358

**Comment by ddkohler** [comment]
@untzag 
> Potentially relevant
https://github.com/conda/menuinst/wiki

**Comment by ddkohler** [comment]
@kameyer226 
> Have you looked at Yarn?

**Comment by kameyer226** [comment]
No,  I only heard of it, and I may have tried something with it a while back but not enough for a real look.

**Comment by ksunden** [comment]
to be clear, @kameyer226, dan was transferring over your own comment there, not asking you the question... that said, yarn is not relevant here, that is a package manager for javascript, not python.

**Comment by kameyer226** [comment]
I wonder if there is an actual term for that kind of a mea culpa
